src/sources/docker_logs/mod.rs

```diff
@@ -102,31 +116,42 @@ pub struct DockerLogsConfig {
     /// containers that match a configured inclusion and are also not excluded will be matched.
     ///
     /// This can be used in conjunction with `include_containers`.
+    #[configurable(metadata(
+        docs::examples = "include_",
+        docs::examples = "include_me_0",
+        docs::examples = "ad08cc418cf9"
+    ))]
     include_containers: Option<Vec<String>>, // Starts with actually, not include
```

Comment:
```suggestion
    /// This can be used in conjunction with `exclude_containers`.
    #[configurable(metadata(
        docs::examples = "include_",
        docs::examples = "include_me_0",
        docs::examples = "ad08cc418cf9"
    ))]
    include_containers: Option<Vec<String>>, // Starts with actually, not include
```

Reply:
Good catch, too much copy pasting! Resolved in 08bada0dc